Package.swift

```diff
@@ -2,6 +2,24 @@
 
 import PackageDescription
 
+var targets: [Target] = []
+#if !os(macOS)
+targets += [
+    .systemLibrary(name: "zlib", path: "sources/zlib", pkgConfig: "zlib"),
+    .target(name: "PNG", dependencies: ["zlib"], path: "sources/png"),
+]
+#else
+targets += [
+    // Without “zlib”.
+    .target(name: "PNG", path: "sources/png"),
+]
+#endif
+targets += [
```

Comment:
Let’s just do the simple thing, get rid of this, and run the test executable in the travis script,
otherwise if you squash the commits, I think this can be merged

Reply:
Will do, unless you beat me to it.